app/views/productors/_suppliers.html.erb

```diff
@@ -0,0 +1,50 @@
+<div class="container">
+  <div class="widget mt-4">
+    <div class="row-wrapper">
+      <% first_category = Productor.category.values.first %>
+      <ul class="nav nav-tabs justify-content-center mb-5" role="tablist">
+        <li class="nav-item">
+          <a class="nav-link active" data-toggle="tab" href="#<%= first_category %>">
+            <%= t "enumerize.productor.category.#{first_category}" %>
+          </a>
+        </li>
+
+        <% Productor.category.values.each do |category| %>
+          <% if category != first_category %>
+            <li class="nav-item">
+              <a class="nav-link" data-toggle="tab" href="#<%= category %>">
+                <%= t "enumerize.productor.category.#{category}" %>
+              </a>
+            </li>
+          <% end %>
+        <% end %>
+      </ul>
+    </div>
```

Comment:
Do all loops in one unique spot, and set the `active` class on the first iteration like this:
```erb
      <ul class="nav nav-tabs justify-content-center mb-5" role="tablist"> 
        <% Productor.category.values.each.with_index do |category, index| %>
            <li class="nav-item">
              <a class="nav-link <%= 'active' if index.zero? %>" data-toggle="tab" href="#<%= category %>">
                <%= t "enumerize.productor.category.#{category}" %>
              </a>
            </li>
        <% end %> 
      </ul> 
```